pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -309,11 +318,12 @@ func (dswp *desiredStateOfWorldPopulator) processPodVolumes(
 			uniquePodName, pod, volumeSpec, podVolume.Name, volumeGidValue)
 		if err != nil {
 			klog.Errorf(
-				"Failed to add volume %q (specName: %q) for pod %q to desiredStateOfWorld. err=%v",
+				"Failed to add volume %s (specName: %s) for pod %q to desiredStateOfWorld: %v",
 				podVolume.Name,
 				volumeSpec.Name(),
 				uniquePodName,
 				err)
+			dswp.desiredStateOfWorld.AddErrorToPod(uniquePodName, err.Error())
```

Comment:
If for some reason if these errors were transient and volume eventually did get added to DSOW.. should we remove the errors from pod if `allVolumesAdded` is true in lin#343?

Reply:
Good idea, I remove errors if `allVolumesAdded`.